src/klein/_app.py

```diff
@@ -247,9 +304,12 @@ def branch_f(instance, request, *a, **kw):
 
             @modified("route '{url}' executor".format(url=url), f)
             def _f(instance, request, *a, **kw):
+                # type: (Any, IRequest, Any, Any) -> KleinRenderable
                 return _call(instance, f, request, *a, **kw)
 
-            _f.segment_count = segment_count
+            _f = cast(KleinRoute, _f)
```

Comment:
`mypy` will flag this an unnecessary when we add type hints to the `@modified` decorator, which I had some trouble with.